src/Timeline.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {COMMON, get} from './constants'
+import theme from './theme'
+import Box from './Box'
+import Flex from './Flex'
+import {Relative} from './Position'
+import classnames from 'classnames'
+
+const Timeline = styled(Flex)`
+  flex-direction: column;
+  ${props =>
+    props.clipSidebar &&
+    `.Timeline-Item:first-child {
+    padding-top: 0;
+  }
+
+  .Timeline-Item:last-child {
+    padding-bottom: 0;
+  }
+`}
+`
+
+Timeline.Item = props => {
+  return <TimelineItemInternal {...props}>{props.children}</TimelineItemInternal>
+}
```

Comment:
What's the purpose of the additional indirection here?

Reply:
Good catch! There isn't any purpose any more, originally I was doing something in here, adding an extra className, maybe? But I must have refactored it all away! Will update to make the styled component be the `Timeline.Item` directly.